### ISSUE TITLE
Material3 / Material component themes

### ISSUE THREAD

**Issue by okwasniewski** [zoontek/react-native-edge-to-edge#38]
### Why it is needed?

Hey! 

Great work on the library! 

I'm working on `react-native-bottom-tabs` which require you to specify Material3 or MaterialComponents theme in styles.xml. 

Would be great to have `Theme.EdgeToEdge.MaterialComponents` and `Theme.EdgeToEdge.Material3` so that users can use both libraries without issues. 

Thanks!

**Comment by zoontek** [comment]
For personal reference: https://okwasniewski.github.io/react-native-bottom-tabs/docs/guides/android-native-styling.html#react-native-community-cli-users

**Comment by zoontek** [comment]
@okwasniewski Edge-to-edge versions of material2 / material3 are available in [1.1.0](https://github.com/zoontek/react-native-edge-to-edge/releases/tag/1.1.0), sorry for the delay!

The good news is that given the fact that the `react-native-bottom-tabs` expo plugin is only setting the `AppTheme` parent theme, there is no need to use it when using the `react-native-edge-to-edge` one 🔥:

```json
{
  "expo": {
    "plugins": [
      [
        "react-native-edge-to-edge",
        { "android": { "parentTheme": "Material3" } }
      ]
    ]
  }
}
```

**Comment by okwasniewski** [comment]
Thank you @zoontek!

I've updated the docs to instruct users to use only one plugin if rn-edge-to-edge is used 🚀 https://github.com/okwasniewski/react-native-bottom-tabs/pull/136
